agr_java_core/src/main/java/org/alliancegenome/es/model/query/Pagination.java

```diff
@@ -75,7 +75,9 @@ private void init(String asc) {
     public void addFieldFilter(FieldFilter fieldFilter, String value) {
         // if mapping exists
         checkIfMappingExists(fieldFilter);
-        fieldFilterValueMap.put(fieldFilter, value);
+        if(value!=null && !value.equals("")) {
```

Comment:
We have been using `if (StringUtils.isNotEmpty(value)) {` for this kind of evaluation. This is fine just giving another suggestion.